VK/VK/FriendPhotoScreen/Cell/PhotoCollectionViewCell.swift

```diff
@@ -41,6 +63,26 @@ final class PhotoCollectionViewCell: UICollectionViewCell {
         firstPhotoImageView.isUserInteractionEnabled = true
     }
 
+    private func getPhotos() {
+        NetworkServiceble.shared.fetchAllPhoto(
+            parametersMap: [
+                Constants.ownerIDName: "\(selectedFriend?.id ?? 0)",
+                Constants.modeParamName: Constants.modeParamValue
```

Comment:
```suggestion
                Constants.modeParamName: Constants.modeParamValue
```
Параметры сетевых запросов в NetworkService